content/en/security/application_security/api-inventory/_index.md

```diff
@@ -6,9 +6,31 @@ further_reading:
   text: "Mitigate the primary risks to API security"
 ---
 
-## Overview
+API security relies on visibility. The biggest failure mode in most programs isn't missed vulnerabilities, it's missed APIs.
 
-[API Security Inventory][7] monitors your API traffic to provide visibility into the security posture of your APIs, including:
+[API Security Inventory][7] monitors your API traffic to provide visibility into the security posture of your APIs.
```

Comment:
This needs to be rewritten as we now monitor more than traffic, e.g. AWS API Gateway configuration, framework configuration and soon we'll also inspect sources.